src/org/traccar/database/NotificationManager.java

```diff
@@ -48,6 +48,10 @@ public NotificationManager(DataManager dataManager) {
     }
 
     public void updateEvent(Event event, Position position) {
+        updateEvent(event, position, false);
```

Comment:
Is there any time when position is not null and we don't want to update it? Possibly not for this pull request, but it would help with handling events in the web app if event and position come in the same message.